src/main/java/mod/gate/features/HudTesting.java

```diff
@@ -1,22 +1,24 @@
 package mod.gate.features;
 
 import com.mojang.blaze3d.systems.RenderSystem;
+import mod.gate.core.config.Config;
 import mod.gate.core.events.Event;
-import mod.gate.core.events.FrameEvent;
+import mod.gate.core.events.RenderEvent;
 import mod.gate.utils.Reference;
 import net.minecraft.client.gui.DrawableHelper;
 import net.minecraft.client.util.math.MatrixStack;
 import net.minecraft.util.Identifier;
 
-public class GuiTesting {
+public class HudTesting {
 
     private final Identifier github = new Identifier(Reference.MODID, "textures/github.png");
 
+    @Config(description = "A random config value that will be written to config file")
+    public final boolean RandomConfigOption = true;
```

Comment:
this isn't used by anything in the mod ?

Reply:
Correct. It’s an example of a config. I still need to make the gui which will build of the description part

Reply:
kk, should prolly just be labeled as an example

Reply:
it's always better to be clear